tests/performance/scale/tests/kube-burner/cluster-density/run-workload.sh

```diff
@@ -177,6 +186,10 @@ function main() {
             kube_burner_path="${2:-}"
             shift
             ;;
+	"--test-name")
+	    test_name="${2:-}"
```

Comment:
The `test_name` variable is not used in the call of `run_workload`.

It would be also good to define it first:
```
local test_name=""
```
(check list at the beginning of `main` function)

Reply:
Fixed